dali/c_api/c_api.cc

```diff
@@ -37,27 +37,23 @@ using dali::AccessOrder;
 using dali::CPUBackend;
 using dali::GPUBackend;
 
-namespace {
-
-bool dali_initialized = false;
-
-/**
- * Maps operator name to the batch size set prior to daliSetExternal... call.
- * Typically, this operator will be BatchSizeProvider.
- * Negative values denote max batch size (default state).
- * Typical usage:
- * auto *batch_size_map = reinterpret_cast<batch_size_map_t *>(handle->batch_size_map);
- */
 using batch_size_map_t = std::unordered_map<std::string /* op_name */, int /* batch_size */>;
+using data_id_map_t = std::unordered_map<std::string /* op_name */, std::string /* data_id */>;
```

Comment:
Why removing documentation of these structures?

Reply:
Perhaps it was added recently and I removed it during rebasing?

Reply:
Restored.